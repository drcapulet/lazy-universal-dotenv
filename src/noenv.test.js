import { getEnvironment } from "."

const snapshotOpts = {}

/* eslint-disable import/no-commonjs */
// We can't use ESM when relying on the fact the the env from the top is correctly respected.
const api = require(".")

test("Defaults to development for NODE_ENV", () => {
  const { raw, stringified, webpack } = api.getEnvironment({ nodeEnv: null })
  expect(raw).toMatchSnapshot(snapshotOpts)
  expect(stringified).toMatchSnapshot(snapshotOpts)
  expect(webpack).toBeDefined()
})
