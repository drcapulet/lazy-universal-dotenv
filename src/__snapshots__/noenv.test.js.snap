// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Defaults to development for NODE_ENV 1`] = `
Object {
  "APP_TEST_OVERRIDE": "original",
  "APP_TEST_PATH": "one/two/three.js",
  "TEST_IGNORE": "123",
}
`;

exports[`Defaults to development for NODE_ENV 2`] = `
Object {
  "APP_TEST_OVERRIDE": "\\"original\\"",
  "APP_TEST_PATH": "\\"one/two/three.js\\"",
  "TEST_IGNORE": "\\"123\\"",
}
`;
